### ISSUE TITLE
Elevate mode have stuck to "As Needed"

### ISSUE THREAD

**Issue by staaronis3** [symless/synergy#5583]
I have some issues with copy-paste functionality, while I am connected to an another pc via Teamviewer.

So, I have tried to change the elevate mode to an other value.

But, after pressing the OK button and go back to settings it is back to "As Needed".

This happens both to server and client.

Server: Windows 10 x64
Client: Windows 7 x64
Both with the 1.8.3-rc1-e36a7ae version of Synergy.

**Comment by nlyan** [comment]
Hi, can you follow the following procedure for me?

1. Hold your Windows key + press r
2. Type regedit and hit OK
3. Navigate using the tree on the left to HKEY_CURRENT_USER -> SOFTWARE -> Synergy -> Synergy

and then give me details of the elevateMode (a REG_SZ record) and elevateModeEnum (a REG_DWORD record) values on the right?

**Comment by staaronis3** [comment]
The values that I see is:
As Needed: false - 0
Always: true - 1
Never: false - 2

**Comment by nlyan** [comment]
@staaronis3 Looks normal. Is the UI still misbehaving?

**Comment by staaronis3** [comment]
It's still misbehaving and I have exactly the same issue as I described it in my first comment.

**Comment by XinyuHou** [comment]
@nlyan 
Have we reproduced this on our side?

**Comment by nlyan** [comment]
@XinyuHou Nope. I thought I'd see if anyone else report the issue with the release candidate

**Comment by anjianshi** [comment]
meet same problem.

Server: ArchLinux x64   1.8.5
Client: Win10 x64 1.8.5-stable-a18eba7

functional is normal, just the UI is incorrect.
